general/v0.1/registration.md

```diff
@@ -27,112 +27,96 @@ administrator. This MAY include but is not limited to the following:
 * Email address and/or other contact data of the adminstrator
 * A password of other means of authentication, so the administrator can
   sign-in again later
-* URL of the fediverse server(s) to be registered
 * Acceptance of terms of service, data processing agreement, and/or privacy policy
 
-A successful registration results in the FASP creating an OAuth 2.0
-application for the fediverse server, including a client identifier (ID) and
-secret. FASP MUST grant the application all the scopes defined here and in
-the FASP specifications of the capabilities the FASP advertises
-to the fediverse server (see section [04: Provider Info](provider_info.md)).
-FASP specifications MAY define exceptions to this rule.
+As part of the registration process the fediverse server administrator
+MUST provide the base URL for FASP interaction of their server.
 
-After registration, FASP MUST present the fediverse server
-administrator a registration token that can be copied into the fediverse server configuration.
+A successful registration results in the FASP creating an Ed25519
+keypair and a client identifier (ID) for the fediverse server.
 
-The registration token is a JSON Web Token (JWT) that MUST include the
-registered claims `iss` and `sub` and the private claim `sec`. It MAY
-contain the registered claim `exp`.
+After registration, FASP MUST make an HTTP `POST` request to the
+fediverse server's `/registration` endpoint.
```

Comment:
I thought about this as well and while it would work, I do not think we _need_ this level of indirection here.

We have the luxury of being able to define whatever API we want relative to the base URL, so simply mandating a static endpoint makes it a lot easier both to specify _and_ to implement. With no obvious downside I could see.